lib/ask/runtime/session.ex

```diff
@@ -321,13 +322,12 @@ defmodule Ask.Runtime.Session do
     end
   end
 
-  defp mode_start(%Session{flow: flow, respondent: respondent, current_mode: %SMSSimulatorMode{}} = session) do
-    case flow |> Flow.step(session.current_mode  |> SessionMode.visitor, :answer, respondent.disposition) do
-      {:end, _, reply} ->
-        {:end, reply, respondent}
-      {:ok, flow, reply} ->
-        {:ok, %{session | flow: flow, respondent: respondent}, reply, current_timeout(session)}
-    end
+  defp mode_start(%Session{current_mode: %SMSSimulatorMode{}} = session) do
+    mode_start_simulation(session)
+  end
+
+  defp mode_start(%Session{current_mode: %MobilewebSimulatorMode{}} = session) do
+    mode_start_simulation(session)
```

Comment:
🤔  I'm not being able to see the benefit of having this `mode_start` with two different signatures if both are calling the same function